3rdparty/halrvv/rvv071/halrvv071.hpp

```diff
@@ -0,0 +1,149 @@
+/*
+ * By downloading, copying, installing or using the software you agree to this license.
+ * If you do not agree to this license, do not download, install,
+ * copy or use the software.
+ *
+ *
+ *                           License Agreement
+ *                For Open Source Computer Vision Library
+ *                        (3-clause BSD License)
+ *
+ * Copyright (C) 2016, NVIDIA Corporation, all rights reserved.
+ * Third party copyrights are property of their respective owners.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ *   * Redistributions of source code must retain the above copyright notice,
+ *     this list of conditions and the following disclaimer.
+ *
+ *   * Redistributions in binary form must reproduce the above copyright notice,
+ *     this list of conditions and the following disclaimer in the documentation
+ *     and/or other materials provided with the distribution.
+ *
+ *   * Neither the names of the copyright holders nor the names of the contributors
+ *     may be used to endorse or promote products derived from this software
+ *     without specific prior written permission.
+ *
+ * This software is provided by the copyright holders and contributors "as is" and
+ * any express or implied warranties, including, but not limited to, the implied
+ * warranties of merchantability and fitness for a particular purpose are disclaimed.
+ * In no event shall copyright holders or contributors be liable for any direct,
+ * indirect, incidental, special, exemplary, or consequential damages
+ * (including, but not limited to, procurement of substitute goods or services;
+ * loss of use, data, or profits; or business interruption) however caused
+ * and on any theory of liability, whether in contract, strict liability,
+ * or tort (including negligence or otherwise) arising in any way out of
+ * the use of this software, even if advised of the possibility of such damage.
+ */
+
+#ifndef OPENCV_HALRVV071_HAL_HPP_INCLUDED
+#define OPENCV_HALRVV071_HAL_HPP_INCLUDED
+
+#if __GNUC__ > 8 || \
+    (__GNUC__ == 8 && (__GNUC_MINOR__ > 4 || \
+                       (__GNUC_MINOR__ == 4 && \
+                        __GNUC_PATCHLEVEL__ > 0)))
+
+#ifndef CV_HAL_RVV_071_ENABLED
+#  if defined(__THEAD_VERSION__) && defined(__riscv_v) && __riscv_v == 7000
+#    define CV_HAL_RVV_071_ENABLED 1
+#  else
+#    define CV_HAL_RVV_071_ENABLED 0
+#  endif
+#endif
+
+#endif /* GCC version check*/
+
+#if CV_HAL_RVV_071_ENABLED
+
+#include "opencv2/core/hal/interface.h"
+
+#include <riscv_vector.h>
+
+#undef cv_hal_cvtBGRtoBGR
+#define cv_hal_cvtBGRtoBGR cvt_hal_BGRtoBGR
```

Comment:
Use more informative and distinguishable name for all functions, maybe add them to the separate namespace, e.g. `cv_hal_rvv::cvtBGRtoBGR`